docs/clients/pulsing_rectangular_input_parser.rst

```diff
@@ -0,0 +1,112 @@
+Pulsing Rectangular Input Parser
+--------------------------------
+This client is used with pulsing modules to use the x and y coordinate commands from the flight controller and converts them into a format that can be used by the pulsing module.
```

Comment:
This pulsing module specific client is used to convert flight controller x/y coordinate commands into a format understood by pulsing modules.